features/contract-state/DisputeParams.tsx

```diff
@@ -23,7 +30,11 @@ const DisputeParams = () => {
     disputeBondPct,
     disputerDisputeRewardPct,
     sponsorDisputeRewardPct,
+    withdrawalLiveness,
   } = empState;
+  const withdrawalLivenessInMinutes = (Number(withdrawalLiveness) / 60).toFixed(
```

Comment:
This withdrawal liveness is already shown in general info?

Reply:
nope. I removed it from general and put it in the dispute section. see the screenshot for how it looks now.